clients/ui/frontend/src/app/pages/modelCatalog/components/HardwareConfigurationFilterToolbar.tsx

```diff
@@ -9,33 +9,81 @@ import {
 } from '@patternfly/react-core';
 import { HelpIcon } from '@patternfly/react-icons';
 import { ModelCatalogContext } from '~/app/context/modelCatalog/ModelCatalogContext';
-import { CatalogPerformanceMetricsArtifact } from '~/app/modelCatalogTypes';
+import { CatalogPerformanceMetricsArtifact, ModelCatalogFilterKey } from '~/app/modelCatalogTypes';
 import { clearAllFilters } from '~/app/pages/modelCatalog/utils/hardwareConfigurationFilterUtils';
+import { hasFiltersApplied } from '~/app/pages/modelCatalog/utils/modelCatalogUtils';
+import {
+  ModelCatalogStringFilterKey,
+  ModelCatalogNumberFilterKey,
+  isLatencyFieldName,
+} from '~/concepts/modelCatalog/const';
 import WorkloadTypeFilter from './globalFilters/WorkloadTypeFilter';
 import HardwareTypeFilter from './globalFilters/HardwareTypeFilter';
 import MinRpsFilter from './globalFilters/MinRpsFilter';
 import MaxLatencyFilter from './globalFilters/MaxLatencyFilter';
+import ModelCatalogActiveFilters from './ModelCatalogActiveFilters';
 
 type HardwareConfigurationFilterToolbarProps = {
   performanceArtifacts: CatalogPerformanceMetricsArtifact[];
 };
 
+/**
+ * Filter keys that are shown on the performance/hardware configuration page.
+ * This is used to determine which filters to show in the active filters chips
+ * and which filters to clear when "Reset all filters" is clicked.
+ */
+const PERFORMANCE_FILTER_KEYS: ModelCatalogFilterKey[] = [
+  ModelCatalogStringFilterKey.USE_CASE,
+  ModelCatalogStringFilterKey.HARDWARE_TYPE,
+  ModelCatalogNumberFilterKey.MIN_RPS,
+];
+
+/**
+ * Gets the active filter keys including any active latency filters from filterData
+ */
+const getActivePerformanceFilterKeys = (
+  filterData: Record<string, unknown>,
+): ModelCatalogFilterKey[] => {
+  const activeLatencyKeys = Object.keys(filterData).filter(
+    (key) => isLatencyFieldName(key) && filterData[key] !== undefined,
+  ) as ModelCatalogFilterKey[];
+
+  return [...PERFORMANCE_FILTER_KEYS, ...activeLatencyKeys];
+};
```

Comment:
We should use the existing type for filterData (`ModelCatalogFilterStates`). Also we don't need isLatencyFieldName here if we invert the logic:

```suggestion
const getActivePerformanceFilterKeys = (
  filterData: ModelCatalogFilterStates,
): ModelCatalogFilterKey[] => {
  const activeLatencyKeys = ALL_LATENCY_FIELD_NAMES.filter((key) => filterData[key] !== undefined);
  return [...PERFORMANCE_FILTER_KEYS, ...activeLatencyKeys];
};
```